### PR TITLE
Added dualserver command

### PR DESCRIPTION
**Pull request by Hamy-os**
Added a dual server command that gives a detailed tutorial on how to have 2 servers since a lot of people ask in the tx admin discord and I'm too lazy to type every time :)

### CODE REVIEW

**Comment by PLOKMJNB** [comment]
!dual already exists

**Comment by Hamy-os** [comment]
> !dual already exists

this one has more detail

File: commands/dualserver.txt [hunk 1]
```diff
@@ -0,0 +1,15 @@
+Step 1: Download the latest server artifact here: https://runtime.fivem.net/artifacts/fivem/build_server_windows/master/?ds
```
Comment (PLOKMJNB) [inline commands/dualserver.txt:1]: Though linux is a courtesy port, its still there

File: commands/dualserver.txt [hunk 2]
```diff
@@ -0,0 +1,15 @@
+Step 1: Download the latest server artifact here: https://runtime.fivem.net/artifacts/fivem/build_server_windows/master/?ds
+Step 2: Make 2 folders in your drive 
+Step 3: Name 1 folder `server1` and the other `server2`
+Step 4: Open the server1 folder and paste in the artifact you downloaded above
+Step 5: Repeat this step for folder named server2
+Step 6: Go in folder server1 and double click fxserver.exe it will open txadmin and follow this tutorial https://www.youtube.com/watch?v=iGfwUCO0RZQ **DO EXACTLY WHAT HE DOES**
```
Comment (PLOKMJNB) [inline commands/dualserver.txt:6]: Why the PlumeESX tutorial? not everyone wants an ESX server

File: commands/dualserver.txt [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+Step 7: Close your fxserver.exe in server1 and close txadmin (close the command prompt)
+Step 8: Go in folder server2 and double click fxserver.exe and follow the txadmin setup, do the same as Step 7 (select plume esx if you want another plume esx server or cfx_default if you want to make a different kind of second server)
+Step 9: Close the fxserver.exe in folder server2 and go to the Start_3372_Default (name may vary depending on what fxversion you have) and right click and click edit
+Step 10: click ctrl+a to select everything then paste this into there
+```@​echo off
+FXServer.exe +set serverProfile "server2" +set txAdminPort "40125"
+pause```
+Step 11: Click ctrl+S and save 
+Step 12: You are done!
```
Comment (PLOKMJNB) [inline commands/dualserver.txt:15]: I don't suppose you have to edit a cfg anywhere

**Review by PLOKMJNB — CHANGES_REQUESTED** [review]

File: commands/dualserver.txt (hunk 1 above)
Comment (Hamy-os) [reply commands/dualserver.txt:1]: true ill add that

File: commands/dualserver.txt (hunk 2 above)
Comment (Hamy-os) [reply commands/dualserver.txt:6]: the second server is non esx its covering multiple bases at once

File: commands/dualserver.txt (hunk 3 above)
Comment (Hamy-os) [reply commands/dualserver.txt:15]: no since recipe deployer does it for you

**Pull request closed without merging**